src/corpus/ui/ChapterViewLine.tsx

```diff
@@ -21,24 +21,33 @@ import TranslationContext from './TranslationContext'
 import { Anchor } from 'transliteration/ui/line-number'
 import Score from './Score'
 import Parallels from './Parallels'
+import { createColumns } from 'transliteration/domain/columns'
+import { numberToUnicodeSubscript } from 'transliteration/ui/SubIndex'
 
 const lineNumberColumns = 1
 const toggleColumns = 3
 const translationColumns = lineNumberColumns + 1
 
 function InterText({
   line,
+  variantNumber,
   colSpan,
 }: {
   line: LineDisplay
+  variantNumber: number
```

Comment:
The component only needs the variant. It would be better to pass it directly, or maybe just the intertext.